### ISSUE TITLE
NullReferenceException when running build

### ISSUE THREAD

**Issue by lilfighterr** [Microsoft/RoomAliveToolkit#77]
Hi,

When I build to a .exe file and run it, all I see is a black screen. The debugger picks up a NullReferenceException on two camera calls:
![image](https://user-images.githubusercontent.com/6673565/38340244-9dec97c8-382f-11e8-8cbb-e0253a29724e.png)

![image](https://user-images.githubusercontent.com/6673565/38340250-aeec6f12-382f-11e8-90f7-9f59eb9ea656.png)

Any suggestions on fixing this?

Thanks

**Comment by thundercarrot** [comment]
Not exactly, but probably some field in some component has not been set. Just to sanity check this, I would follow the Unity tutorial to the letter and see if you get the same error (I'm presuming here that this is not the tutorial-- correct me if wrong).

**Comment by lilfighterr** [comment]
I followed the Unity tutorial exactly and also retried it again from scratch. Still the same problem. I initially had the building problem mentioned in https://github.com/Microsoft/RoomAliveToolkit/issues/75 and fixed it by following jjhartmann's comment.

**Comment by ahmedassal** [comment]
I have the exact same problem after doing the exact same steps as @lilfighterr

**Comment by lilfighterr** [comment]
@ahmedassal I fixed this by moving some of the code that is giving the problem into the Awake() function so that it initializes first. Seems to be working with no problems so far.
